File: solutions/java/two-fer/1/src/main/java/Twofer.java
public class Twofer {
    public String twofer(String name) {
        // If name is null or empty, default to "you"
        if (name == null || name.isEmpty()) {
            return "One for you, one for me.";
        }
        
        // Otherwise, use the provided name
        return "One for " + name + ", one for me.";
    }
}

File: solutions/go/two-fer/1/two_fer.go
package twofer

import "fmt"

// ShareWith returns a string sharing a cookie with the given name, or "you" if the name is empty.
func ShareWith(name string) string {
	if name == "" {
		name = "you"
	}
	return fmt.Sprintf("One for %s, one for me.", name)
}